src/main/webapp/__tests__/createGameElements.test.js

```diff
@@ -0,0 +1,128 @@
+    for (var i = 0; i < 4; i += 2) {
+      answersStr += `
+      <!-- The first line -->
+      <div class="mdl-grid">
+        <div class="mdl-cell mdl-cell--1-col"></div>
+        <div class="mdl-cell mdl-cell--5-col">
+          <!-- First answer -->
+          <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+            <input class="mdl-textfield__input" type="text" id="game-correct-answer" name="answer` + i + `"> <label class="mdl-textfield__label" > Answer ` + (i+1) + `</label>
+          </div>
+    
+          <!-- Check if it's correct -->
+          <label class="mdl-checkbox mdl-js-checkbox mdl-js-ripple-effect" for="question` + num + `checkbox` + i + `">
+            <input type="checkbox" id="question` + num + `checkbox` + i + `" class="mdl-checkbox__input" name="correct` + i + `">
+            <span class="mdl-checkbox__label">Correct</span>
+          </label>
+        </div>
+        <div class="mdl-cell mdl-cell--5-col">
+          <!-- Second answer -->
+          <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+            <input class="mdl-textfield__input" type="text" id="game-correct-answer" name="answer` + (i+1) + `"> <label class="mdl-textfield__label" > Answer ` + (i+2) + `</label>
+          </div>
+    
+          <!-- Check if it's correct -->
+          <label class="mdl-checkbox mdl-js-checkbox mdl-js-ripple-effect" for="question` + num + `checkbox` + (i+1) + `">
+            <input type="checkbox" id="question` + num + `checkbox` + (i+1) + `" class="mdl-checkbox__input" name="correct` + (i+1) + `">
+            <span class="mdl-checkbox__label">Correct</span>
+          </label>
+        </div>
+        <div class="mdl-cell mdl-cell--1-col"></div>
+      </div>
+      `
+    }
+    document.body.innerHTML = "<div id='multipleChoiceDiv'>" + answersStr + "<div>";
+
+    // Call the function under test
+    const expectedmultipleChoiceDiv = multipleChoiceGrid(num);
+    expect(document.getElementById('multipleChoiceDiv').innerHTML).toContain(expectedmultipleChoiceDiv.innerHTML);
+  });
+  test("Is the title grid appearing", () => {
+    var title = "Hello this is a test!!!";
+
+    document.body.innerHTML =  "<div id='titleGrid'>" + '<div class="mdl-grid">' +
+    '  <div class="mdl-cell mdl-cell--3-col"></div>' +
+      '<div class="mdl-cell mdl-cell--6-col">' +
+      '  <h3>' + title + '</h3>' + 
+        '<div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">' +
+          '<input class="mdl-textfield__input" type="text" name="questionTitle"> <label class="mdl-textfield__label" >Question Title</label>' +
+        '</div>' +
+      '</div>' +
+      '<div class="mdl-cell mdl-cell--3-col"></div>' +
+    '</div>' + '</div>';
+
+    const expectedTitleDiv = titleGrid(title);
+    expect(document.getElementById('titleGrid').innerHTML).toContain(expectedTitleDiv.innerHTML);
+  });
+
+  test("Randomly generated question type", () => {
```

Comment:
I think the 50 random question set up is unnecessary. Just test properties on the output of `questionElement(0, 'q1', true)`, and `questionElement(0, 'q1', false)`